BridgeAppSDK/Resources/ProfileItems.json

```diff
@@ -0,0 +1,46 @@
+{
+    "items":    [
+                 {
+                 "profileKey": "participantFirstName",
+                 "classType": "StudyParticipantProfileItem",
+                 "sourceKey": "firstName",
+                 "itemType": "String"
+                 },
+                 {
+                 "profileKey": "participantLastName",
+                 "classType": "StudyParticipantProfileItem",
+                 "sourceKey": "lastName",
+                 "itemType": "String"
+                 },
+                 {
+                 "profileKey": "participantEmail",
```

Comment:
I worry participantUsername would be as confusing since we used to call that signUp field username, but was deprecated in favor of email since it must be an email address...